homeassistant/components/homeassistant_yellow/config_flow.py

```diff
@@ -33,18 +44,32 @@
 )
 
 
-class HomeAssistantYellowConfigFlow(ConfigFlow, domain=DOMAIN):
+class HomeAssistantYellowConfigFlow(
+    firmware_config_flow.BaseFirmwareConfigFlow, domain=DOMAIN
+):
     """Handle a config flow for Home Assistant Yellow."""
 
     VERSION = 1
+    MINOR_VERSION = 2
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Instantiate config flow."""
+        super().__init__(*args, **kwargs)
+
+        self._device = RADIO_DEVICE
```

Comment:
`self._device` is the device that the firmware config flow will be communicating with. Since it's the Yellow, we know what the device is in advance (it's always `/dev/ttyAMA1`).